homeassistant/components/home_connect/translations/sensor.en.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The `translations` folder isn't part of this repository anymore, this file can be removed.